### ISSUE TITLE
Errors while going through the tutorial

### ISSUE THREAD

**Issue by ronensc** [caikit/website#22]
I followed the [Try it out!](https://caikit.github.io/website/docs/tutorial_appdev.html#5-start-the-caikit-runtime) tutorial and encountered errors when running `python3 start_runtime.py` in section `5. Start the Caikit runtime`.

Caikit version: `0.26.0`

To resolve the issue, I had to install the following packages using the command:
```
pip install grpcio-health-checking grpcio-reflection py-grpc-prometheus
```

Without these packages, I encountered the following errors:

1. ModuleNotFoundError: No module named 'grpc_health'
```
Traceback (most recent call last):
  File "/home/ronen/caikit-demo/text-sentiment/start_runtime.py", line 32, in <module>
    from caikit.runtime import grpc_server
  File "/home/ronen/caikit-demo/text-sentiment/.venv/lib/python3.9/site-packages/caikit/runtime/grpc_server.py", line 20, in <module>
    from grpc_health.v1 import health, health_pb2_grpc
ModuleNotFoundError: No module named 'grpc_health'
```

2. ModuleNotFoundError: No module named 'grpc_reflection'
```
Traceback (most recent call last):
  File "/home/ronen/caikit-demo/text-sentiment/start_runtime.py", line 32, in <module>
    from caikit.runtime import grpc_server
  File "/home/ronen/caikit-demo/text-sentiment/.venv/lib/python3.9/site-packages/caikit/runtime/grpc_server.py", line 21, in <module>
    from grpc_reflection.v1alpha import reflection
ModuleNotFoundError: No module named 'grpc_reflection'
```

3. ModuleNotFoundError: No module named 'py_grpc_prometheus'
```
Traceback (most recent call last):
  File "/home/ronen/caikit-demo/text-sentiment/start_runtime.py", line 32, in <module>
    from caikit.runtime import grpc_server
  File "/home/ronen/caikit-demo/text-sentiment/.venv/lib/python3.9/site-packages/caikit/runtime/grpc_server.py", line 22, in <module>
    from py_grpc_prometheus.prometheus_server_interceptor import PromServerInterceptor
ModuleNotFoundError: No module named 'py_grpc_prometheus'
```

Please let me know if you have any further questions or if you would like additional context.

**Comment by dmuelle** [comment]
@hickeyma - would you review the error that @ronensc encountered and recommend the best way to satisfy the missing dependency in the context of this tutorial? Should it be handled in step 5. or earlier?

**Comment by krook** [comment]
Hey folks, came here to report something similar. I had to install the following for it to work:

```sh
pip3 install grpcio 
pip3 install grpcio-health-checking
pip3 install grpcio-reflection
pip3 install py-grpc-prometheus
```

As one going through the tutorial, I would expect these to be in the `requirements.txt` file.

**Comment by gabe-l-hart** [comment]
## TLDR

```sh
pip install caikit[runtime-grpc]
```

## Details

Hi folks! Thanks for raising this issue. The root of the problem is that the tutorial was not updated to reflect the introduction of optional dependencies ([here](https://github.com/caikit/caikit/blob/main/pyproject.toml#L44)). This was added in `caikit==0.9.0` ([here](https://github.com/caikit/caikit/releases/tag/v0.9.0)). I'll submit a fix for the tutorial, but in the meantime you can use the TLDR above to get around it (with the officially compatible version ranges).
